cluster-up-example

```diff
@@ -18,7 +18,13 @@ oc_password=developer
 #Digger Variables
 digger_project_name=digger
 digger_platforms="android,ios"
-
+#proxy_url=127.0.0.1
```

Comment:
actually you can ignore the above. I see you are using proxy_url here to specify the host. Could I suggest a change in the name of the variable to proxy_host as we are using proxy_url in rhmap ansible in the way I have described above.